app/graphql/types/category.rb

```diff
@@ -0,0 +1,50 @@
+class Types::Category < Types::BaseObject
+  description 'Information about a specific Category'
+
+  field :id, ID, null: false
+
+  field :title, String,
+    null: false,
+    description: 'The name of the category.'
+
+  field :description, String,
+    null: true,
+    description: 'A description of the category.'
+
+  field :slug, String,
+    null: false,
+    description: 'The URL-friendly identifier of this Category.'
+
+  field :anidb_id, Integer,
```

Comment:
Can we make this type `ID` instead? Just so you know it doesnt indicate a primary key concept so totally kosher to have multiple `ID` types.

Reply:
oh, totally didn't know that! Thank you!